ietf/templates/doc/document_info.html

```diff
@@ -351,6 +351,9 @@
                 {% for review_assignment in review_assignments %}
                     {% include "doc/review_assignment_summary.html" with current_doc_name=doc.name current_rev=doc.rev %}
                 {% endfor %}
+                {% for review_request in review_requests %}
+                    {% include "doc/review_request_summary.html" with current_doc_name=doc.name current_rev=doc.rev %}
```

Comment:
```suggestion
                    {% include "doc/review_request_summary.html" with current_doc_name=doc.name current_rev=doc.rev only %}
```

Reply:
I have no idea what that change would do so can't really comment on that.

Reply:
it passes only the current_doc_name and current_rev to the review_request_summary template instead of the full context available to the template you are calling `{% include` from.

Reply:
but I'm really confused - review_request_summary.html only uses `review_request`, why are you passing current_doc_name and current_rev to it?

Reply:
Because it is cut & paste from the above where it was giving same parameters to the doc/review_assignment_symmary.html.

I copied the review_assignment_summary.html as a base for the review_request_summary.html and at that point I did not know whether the lines using current_doc stays in or not.

In the end I did not need them as they are used to remove the version number from the line telling which version the review was done for in case it is current revision and document.

So most likely I should simply change it to 

{% include "doc/review_request_summary.html" with review_request only %}

or do I need to use review_request=review_request before only?

Reply:
you need to use `review_request=review_request`

Reply:
Ok, now this should be done. I also did the same for the review_assignment_summary include.